content/nginxaas-azure/changelog.md

```diff
@@ -13,6 +13,17 @@ To see a list of currently active issues, visit the [Known issues]({{< ref "/ngi
 
 To review older entries, visit the [Changelog archive]({{< ref "/nginxaas-azure/changelog-archive" >}}) section.
 
+## May 09, 2025
+
+- {{% icon-feature %}} **NGINXaaS is now running NGINX Plus Release 33 (R33) in the Stable Upgrade Channel**
+
+   NGINXaaS for Azure deployments using the **Stable** [Upgrade Channel]({{< ref "/nginxaas-azure/quickstart/upgrade-channels.md" >}}) have now been automatically upgraded to  [NGINX Plus Release 33 (R33)]({{< ref "/nginx/releases.md#nginxplusrelease-33-r33" >}}). This upgrade also includes updates to the following NGINX Plus modules:
+   - nginx-plus-module-njs
+
+   For entitlement and visibility policy, check [the FAQs]({{< ref "/nginxaas-azure/faq.md" >}}) to address any additional questions you might have.
```

Comment:
```suggestion
   For entitlement and visibility policy, check [the FAQs]({{< ref "/nginxaas-azure/faq.md" >}}) to address any additional questions you might have.
   
```
We are trying to move the content of the FAQ to a more appropriate locations and remove this doc. Not sure if linking to it is the best practice, knowing that.

Reply:
Planning to merge this doc today, it looks like FAQ is still in the same location. Should I keep this ref link?

Reply:
Please change this to a section outside of the FAQ or remove the sentence